src/manager/agent.manager.service.ts

```diff
@@ -8,6 +8,7 @@ import { DockerService } from './docker.service';
 import { IAgentManager } from './agent.manager.interface';
 import { AgentConfig } from './agent.config';
 import { K8sService } from './k8s.service';
+import { agent } from 'supertest';
```

Comment:
I think this can be removed

Reply:
Im a little surprised lint didnt pick that up.  import removed.